installations/Common/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,111 @@
+{
+  "images" : [
```

Comment:
can use the shared Assets(https://github.com/firebase/quickstart-ios/tree/master/shared) instead of a new folder.

Reply:
Thanks! Done.